plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -1083,6 +1083,10 @@ jQuery( function ( $ ) {
 				} else {
 					wc_meta_boxes_product_variations_ajax.unblock();
 				}
+
+				window.wcTracks.recordEvent( 'product_variations_buttons', {
+					action: 'remove_variation',
+				} );
 			}
```

Comment:
I don't seem to see this event triggered ever (looking at filtered network tab).